File: SyntaxErrors/sum.py
# Python : Calculate the sum of an array
def calculate_sum(arr):
    total = 0  # Error: 'o' instead of '0'
    for num in arr:  # Error: Missing colon ':'
        total += num
    return total

numbers = [1, 2, 3, 4, 5]
result = calculate_sum(numbers)
print("Sum in Python:", result)

File: SyntaxErrors/sum.js
// JavaScript : Calculate the sum of an array
function calculateSum(arr) {
    let total = 0;  // Error: 'o' instead of '0'
    for (let num of arr){  // Error: Missing curly brace '{'
        total += num;
    }
    return total;
}
 
let numbers = [1, 2, 3, 4, 5];
let result = calculateSum(numbers);  // Error: Space in function name
console.log("Sum in JavaScript:", result);